src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/validator/GenerateUrlRequestValidatorImplTest.java

```diff
@@ -71,16 +74,28 @@ public void setup() {
 
         Mockito
                 .doReturn(false)
-                .when(courtServiceMock).isValidCourt(ArgumentMatchers.argThat(x -> x.getCourtId().equals(COURT_ID_2)));
+                .when(courtServiceMock)
+                .isValidCourt(ArgumentMatchers.argThat(x -> x.getCourtId().equals(COURT_ID_2)));
+
+
+        Mockito
+                .doReturn(true)
+                .when(courtServiceMock)
+                .isValidCourtFileNumber(ArgumentMatchers.argThat(x -> x.getFileNumber().equals(FILE_NUMBER_SUCCESS)));
+
+        Mockito
+                .doReturn(false)
+                .when(courtServiceMock)
+                .isValidCourtFileNumber(ArgumentMatchers.argThat(x -> x.getFileNumber().equals(FILE_NUMBER_ERROR)));
 
 
         sut = new GenerateUrlRequestValidatorImpl(submissionService, courtServiceMock);
 
     }
 
     @Test
-    @DisplayName("ok: without error should return a notification without error")
-    public void withoutErrorShouldReturnNoError() {
+    @DisplayName("ok: new submission without error should return a notification without error")
```

Comment:
are the display names with "ok" that return an error actually ok? I assume you mean return 200?

Reply:
yes and no. at the method level it's not returning a 200, but at the controller level yeah